userapi/internal/device_list_update.go

```diff
@@ -431,6 +460,7 @@ func (u *DeviceListUpdater) worker(ch chan spec.ServerName, workerID int) {
 		_, exists := retries[serverName]
 		retriesMu.Unlock()
 		if exists {
+			deviceListUpdaterBackpressure.With(prometheus.Labels{"worker_id": strconv.Itoa(workerID)}).Dec()
```

Comment:
I am unconvinced that this is going to track counts correctly.  It is incremented for every call to `notifyWorkers(userID)` but only decremented under very certain scenarios (when not full, etc).

Reply:
See above.